### PR TITLE
Remove Docker Image Build

### PR DESCRIPTION
**Pull request by jamesmcclain**
The docker image build has been moved [here](http://github.com/geodocker/geodocker-jupyter-geopyspark).

### CODE REVIEW

**Comment by jpolchlo** [comment]
Working on bringing this up to date with current master.  Notebook runs, but something odd is happening in display, with significant computation at the point of serving each tile.  Possibly related to the calculation of histograms?  I'm working on it.

File: docker/kernels/geonotebook/kernel.json [hunk 1]
```diff
@@ -1,20 +0,0 @@
-{
```
Comment (echeipesh) [inline docker/kernels/geonotebook/kernel.json]: Pointing out an inconsistency here:
In the master we've been able to use this `kernel.json` on both local and yarn case
where is the `geodocker-jupyter-geonotebook` repo still has two kernels
https://github.com/geodocker/geodocker-jupyter-geopyspark/tree/master/kernels

Comment (jamesmcclain) [inline docker/kernels/geonotebook/kernel.json]: Thank you, I have a branch that corrects that

Comment (jamesmcclain) [inline docker/kernels/geonotebook/kernel.json]: https://github.com/geodocker/geodocker-jupyter-geopyspark/blob/master/kernels/geonotebook/kernel.json

**Comment by jamesmcclain** [comment]
I think this is ready to be merged.

**Pull request merged**